### ISSUE TITLE
Does this work with a local Mosquitto broker?

### ISSUE THREAD

**Issue by Janstander2011** [ckrey/MQTTInspector#86]
For the life of me I can’t get my subs to come from the broker.  In “c” mode I see where the subscribe is sent and the  every minute or so of the ping back and forth between the app and the broker... I just can’t understand what I’m doing wrong. I don’t have any authentication on my broker.

**Comment by jpmens** [comment]
There's not much anybody can do for you with that information ... :)

Check your Mosquitto logs (typically `/var/log/mosquitto/mosquitto.log`); you should see your MQTTinspector connect and subscribe (if it does); double-check your subscriptions, etc.

The program generally works very well here!

**Comment by Janstander2011** [comment]
Ok I’ll check on the broker side. Thanks for the tip im the right direction.

**Comment by Janstander2011** [comment]
So I reviewed the Mosquitto log and it shows an invalid qos in subscription command 

I changed the sub settings under mqtt 5.0 and set them both to 0 and turned off the retain as published flag. 

Now I am getting the subs data coming to the app. 

Obviously I have more to learn on how to get the most out of your app, a user guide would be helpful. 

Thanks for the point in the right direction.
